internal/internal_event_handlers.go

```diff
@@ -1234,94 +1204,94 @@ func (weh *workflowExecutionEventHandlerImpl) handleChildWorkflowExecutionStarte
+func (weh *workflowExecutionEventHandlerImpl) handleChildWorkflowExecutionCanceled(event *m.HistoryEvent) {
 	attributes := event.ChildWorkflowExecutionCanceledEventAttributes
 	childWorkflowID := attributes.WorkflowExecution.GetWorkflowId()
 	decision := weh.decisionsHelper.handleChildWorkflowExecutionCanceled(childWorkflowID)
 	childWorkflow := decision.getData().(*scheduledChildWorkflow)
 	if childWorkflow.handled {
-		return nil
+		return
 	}
 	details := newEncodedValues(attributes.Details, weh.GetDataConverter())
 	err := NewCanceledError(details)
 	childWorkflow.handle(nil, err)
-	return nil
+	return
 }
 
-func (weh *workflowExecutionEventHandlerImpl) handleChildWorkflowExecutionTimedOut(event *m.HistoryEvent) error {
+func (weh *workflowExecutionEventHandlerImpl) handleChildWorkflowExecutionTimedOut(event *m.HistoryEvent) {
 	attributes := event.ChildWorkflowExecutionTimedOutEventAttributes
 	childWorkflowID := attributes.WorkflowExecution.GetWorkflowId()
 	decision := weh.decisionsHelper.handleChildWorkflowExecutionClosed(childWorkflowID)
 	childWorkflow := decision.getData().(*scheduledChildWorkflow)
 	if childWorkflow.handled {
-		return nil
+		return
 	}
 	err := NewTimeoutError(attributes.GetTimeoutType())
 	childWorkflow.handle(nil, err)
 
-	return nil
+	return
 }
 
-func (weh *workflowExecutionEventHandlerImpl) handleChildWorkflowExecutionTerminated(event *m.HistoryEvent) error {
+func (weh *workflowExecutionEventHandlerImpl) handleChildWorkflowExecutionTerminated(event *m.HistoryEvent) {
 	attributes := event.ChildWorkflowExecutionTerminatedEventAttributes
 	childWorkflowID := attributes.WorkflowExecution.GetWorkflowId()
 	decision := weh.decisionsHelper.handleChildWorkflowExecutionClosed(childWorkflowID)
 	childWorkflow := decision.getData().(*scheduledChildWorkflow)
 	if childWorkflow.handled {
-		return nil
+		return
 	}
 	err := newTerminatedError()
 	childWorkflow.handle(nil, err)
 
-	return nil
+	return
 }
 
 func (weh *workflowExecutionEventHandlerImpl) handleUpsertWorkflowSearchAttributes(event *m.HistoryEvent) {
 	weh.updateWorkflowInfoWithSearchAttributes(event.UpsertWorkflowSearchAttributesEventAttributes.SearchAttributes)
 }
 
-func (weh *workflowExecutionEventHandlerImpl) handleRequestCancelExternalWorkflowExecutionInitiated(event *m.HistoryEvent) error {
+func (weh *workflowExecutionEventHandlerImpl) handleRequestCancelExternalWorkflowExecutionInitiated(event *m.HistoryEvent) {
 	// For cancellation of child workflow only, we do not use cancellation ID
 	// for cancellation of external workflow, we have to use cancellation ID
 	attribute := event.RequestCancelExternalWorkflowExecutionInitiatedEventAttributes
 	workflowID := attribute.WorkflowExecution.GetWorkflowId()
 	cancellationID := string(attribute.Control)
 	weh.decisionsHelper.handleRequestCancelExternalWorkflowExecutionInitiated(event.GetEventId(), workflowID, cancellationID)
-	return nil
+	return
```

Comment:
I see why this happened by it's a bit weird to have a noop return in the end (applies several places).

Reply:
Removed